src/applications/personalization/profile360/components/DowntimeBanner.jsx

```diff
@@ -1,22 +1,19 @@
 import React from 'react';
 import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
-import { externalServiceStatus } from '../../../../platform/monitoring/DowntimeNotification';
+import { externalServiceStatus } from 'platform/monitoring/DowntimeNotification';
 
 function DowntimeBanner({ downtime, section }) {
   return (
     <AlertBox
       status="warning"
       isVisible
+      headline={`We can’t show your ${section} information right now.`}
       content={
-        <div>
-          <h3>We can’t show your {section} information right now.</h3>
-          <p>
-            We’re sorry. The system that handles {section} information is down
-            for maintenance right now. We hope to be finished with our work by{' '}
-            {downtime.startTime.format('MMMM Do')},{' '}
-            {downtime.endTime.format('LT')} Please check back soon.
-          </p>
-        </div>
+        <p>
+          We’re sorry. The system that handles {section} information is down for
+          maintenance right now. We hope to be finished with our work by{' '}
+          {downtime.endTime.format('MMMM Do, LT')} Please check back soon.
```

Comment:
I don't know why we were previously using the month and date of the _start_ date rather than _end_ date. Seems like that must have been done in error? Am I missing something, @ncksllvn ?

Reply:
Yeah, that looks like an oversight to me. It was an added in a rush IIRC.